
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://morrisanimalfoundation.github.io/grGWAS/5.pheno/">
      
      
        <link rel="prev" href="../4.qc_filter/">
      
      
        <link rel="next" href="../6.heritability/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.4">
    
    
      
        <title>5. Study design - grGWAS docs</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
   <link href="../assets/stylesheets/glightbox.min.css" rel="stylesheet"/><style>
        html.glightbox-open { overflow: initial; height: 100%; }
        .gslide-title { margin-top: 0px; user-select: text; }
        .gslide-desc { color: #666; user-select: text; }
        .gslide-image img { background: white; }
        
            .gscrollbar-fixer { padding-right: 15px; }
            .gdesc-inner { font-size: 0.75rem; }
            body[data-md-color-scheme="slate"] .gdesc-inner { background: var(--md-default-bg-color);}
            body[data-md-color-scheme="slate"] .gslide-title { color: var(--md-default-fg-color);}
            body[data-md-color-scheme="slate"] .gslide-desc { color: var(--md-default-fg-color);}
            </style> <script src="../assets/javascripts/glightbox.min.js"></script></head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#51-explore-phenotypes" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="grGWAS docs" class="md-header__button md-logo" aria-label="grGWAS docs" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            grGWAS docs
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              5. Study design
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69Z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12c0-2.42-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/morrisanimalfoundation/grGWAS" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    morrisanimalfoundation/grGWAS
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../1.install/" class="md-tabs__link">
          
  
    
  
  Tutorial

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../ext_docs/health_concerns/" class="md-tabs__link">
          
  
    
  
  Bulletin

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="grGWAS docs" class="md-nav__button md-logo" aria-label="grGWAS docs" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    grGWAS docs
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/morrisanimalfoundation/grGWAS" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    morrisanimalfoundation/grGWAS
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tutorial
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Tutorial
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../1.install/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    1. Environment Setup
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../2.Input_files/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    2. Input Description
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../3.prep_arrays/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3. Prep Array Sets
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../4.qc_filter/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4. QC & filtration
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    5. Study design
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    5. Study design
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#51-explore-phenotypes" class="md-nav__link">
    <span class="md-ellipsis">
      5.1. Explore phenotypes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#511-what-is-your-phenotype-of-interest" class="md-nav__link">
    <span class="md-ellipsis">
      5.1.1. What is your phenotype of interest?
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#512-identify-affected-cases-until-last-year" class="md-nav__link">
    <span class="md-ellipsis">
      5.1.2. Identify affected cases until last year
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#513-count-the-of-cases-of-each-phenotype" class="md-nav__link">
    <span class="md-ellipsis">
      5.1.3. Count the of cases of each phenotype
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#52-sample-selection" class="md-nav__link">
    <span class="md-ellipsis">
      5.2. Sample selection
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#521-select-cases-of-a-target-phenotype" class="md-nav__link">
    <span class="md-ellipsis">
      5.2.1 Select "cases" of a target phenotype
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#522-check-for-co-existing-conditions" class="md-nav__link">
    <span class="md-ellipsis">
      5.2.2 Check for co-existing conditions
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#523-identify-individuals-to-be-excluded-from-controls" class="md-nav__link">
    <span class="md-ellipsis">
      5.2.3. Identify individuals to be excluded from controls
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#524-remove-the-excluded-samples-from-the-genotyping-dataset-include-the-oldies-if-not-testing-a-phenotype-with-high-mortality-in-young-age" class="md-nav__link">
    <span class="md-ellipsis">
      5.2.4. Remove the excluded samples from the genotyping dataset (include the oldies if not testing a phenotype with high mortality in young age)
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#525-exclusion-of-1st-degree-relatives" class="md-nav__link">
    <span class="md-ellipsis">
      5.2.5. Exclusion of 1st degree relatives
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#53-generate-the-phenotype-file" class="md-nav__link">
    <span class="md-ellipsis">
      5.3. Generate the phenotype file
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#54-linkage-disequilibrium-ld-analysis" class="md-nav__link">
    <span class="md-ellipsis">
      5.4. Linkage disequilibrium (LD) analysis
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#541-ld-estimation" class="md-nav__link">
    <span class="md-ellipsis">
      5.4.1 LD estimation
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#542-ld-pruning" class="md-nav__link">
    <span class="md-ellipsis">
      5.4.2 LD pruning
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#543-ld-re-evaluation" class="md-nav__link">
    <span class="md-ellipsis">
      5.4.3 LD re-evaluation
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#55-pca-to-identify-population-admixture-and-to-assess-distribution-of-cases" class="md-nav__link">
    <span class="md-ellipsis">
      5.5. PCA to identify population admixture and to assess distribution of cases
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#551-calculation-of-pca" class="md-nav__link">
    <span class="md-ellipsis">
      5.5.1. Calculation of PCA
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#552-assess-possible-batch-effect" class="md-nav__link">
    <span class="md-ellipsis">
      5.5.2. Assess possible batch effect
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#553-assess-distribution-of-cases-in-pca" class="md-nav__link">
    <span class="md-ellipsis">
      5.5.3 Assess distribution of cases in PCA
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../6.heritability/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6. Heritability
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../7.gwas/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7. GWAS Analysis
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Bulletin
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Bulletin
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../ext_docs/health_concerns/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Health concerns
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../ext_docs/gwas_studies/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Previous GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../ext_docs/sample_distance/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sample-distance
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../ext_docs/admixture_analysis/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../ext_docs/GREML_analysis/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GREML analysis
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#51-explore-phenotypes" class="md-nav__link">
    <span class="md-ellipsis">
      5.1. Explore phenotypes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#511-what-is-your-phenotype-of-interest" class="md-nav__link">
    <span class="md-ellipsis">
      5.1.1. What is your phenotype of interest?
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#512-identify-affected-cases-until-last-year" class="md-nav__link">
    <span class="md-ellipsis">
      5.1.2. Identify affected cases until last year
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#513-count-the-of-cases-of-each-phenotype" class="md-nav__link">
    <span class="md-ellipsis">
      5.1.3. Count the of cases of each phenotype
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#52-sample-selection" class="md-nav__link">
    <span class="md-ellipsis">
      5.2. Sample selection
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#521-select-cases-of-a-target-phenotype" class="md-nav__link">
    <span class="md-ellipsis">
      5.2.1 Select "cases" of a target phenotype
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#522-check-for-co-existing-conditions" class="md-nav__link">
    <span class="md-ellipsis">
      5.2.2 Check for co-existing conditions
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#523-identify-individuals-to-be-excluded-from-controls" class="md-nav__link">
    <span class="md-ellipsis">
      5.2.3. Identify individuals to be excluded from controls
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#524-remove-the-excluded-samples-from-the-genotyping-dataset-include-the-oldies-if-not-testing-a-phenotype-with-high-mortality-in-young-age" class="md-nav__link">
    <span class="md-ellipsis">
      5.2.4. Remove the excluded samples from the genotyping dataset (include the oldies if not testing a phenotype with high mortality in young age)
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#525-exclusion-of-1st-degree-relatives" class="md-nav__link">
    <span class="md-ellipsis">
      5.2.5. Exclusion of 1st degree relatives
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#53-generate-the-phenotype-file" class="md-nav__link">
    <span class="md-ellipsis">
      5.3. Generate the phenotype file
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#54-linkage-disequilibrium-ld-analysis" class="md-nav__link">
    <span class="md-ellipsis">
      5.4. Linkage disequilibrium (LD) analysis
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#541-ld-estimation" class="md-nav__link">
    <span class="md-ellipsis">
      5.4.1 LD estimation
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#542-ld-pruning" class="md-nav__link">
    <span class="md-ellipsis">
      5.4.2 LD pruning
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#543-ld-re-evaluation" class="md-nav__link">
    <span class="md-ellipsis">
      5.4.3 LD re-evaluation
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#55-pca-to-identify-population-admixture-and-to-assess-distribution-of-cases" class="md-nav__link">
    <span class="md-ellipsis">
      5.5. PCA to identify population admixture and to assess distribution of cases
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#551-calculation-of-pca" class="md-nav__link">
    <span class="md-ellipsis">
      5.5.1. Calculation of PCA
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#552-assess-possible-batch-effect" class="md-nav__link">
    <span class="md-ellipsis">
      5.5.2. Assess possible batch effect
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#553-assess-distribution-of-cases-in-pca" class="md-nav__link">
    <span class="md-ellipsis">
      5.5.3 Assess distribution of cases in PCA
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  
    
      
    
    <a href="https://github.com/morrisanimalfoundation/grGWAS/tree/main/docs/5.pheno.md" title="View source of this page" class="md-content__button md-icon">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 18c.56 0 1 .44 1 1s-.44 1-1 1-1-.44-1-1 .44-1 1-1m0-3c-2.73 0-5.06 1.66-6 4 .94 2.34 3.27 4 6 4s5.06-1.66 6-4c-.94-2.34-3.27-4-6-4m0 6.5a2.5 2.5 0 0 1-2.5-2.5 2.5 2.5 0 0 1 2.5-2.5 2.5 2.5 0 0 1 2.5 2.5 2.5 2.5 0 0 1-2.5 2.5M9.27 20H6V4h7v5h5v4.07c.7.08 1.36.25 2 .49V8l-6-6H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h4.5a8.15 8.15 0 0 1-1.23-2Z"/></svg>
    </a>
  


  <h1>5. Study design</h1>

<p>The phenotype of interest will define the GWAS design. Some phenotypes are quantitative like body mass index or fasting blood glucose while others are qualitative, binary phenotypes (e.g., Case/Control studies). Each type requires special preparation of the input data, proper selection of the statistical tests and mindful interpretation of the final results. This tutorial is an example for a Case/Control study.  </p>
<p>To start, the cases and controls must be selected based on the inclusion and exclusion criteria for the project. Accordingly, a subset of the genotypic and phenotypic data should be established. Then, the possible correlation of the phenotype of interest should assessed with the genetic principal components in the studied population.  The possible covariates that might affect the phenotype of interest must also be identified, but that will be done during the assessment of heritability of the phenotype in the studied population. </p>
<h2 id="51-explore-phenotypes">5.1. Explore phenotypes</h2>
<h2 id="511-what-is-your-phenotype-of-interest">5.1.1. What is your phenotype of interest?</h2>
<p>As explained earlier, the GRLS phenotypic data is organized in modular data tables. Many clinical conditions are organized in tables by the affectd tissue. For example, several cutaneous disorders are grouped in a table named "conditions_skin.csv". Let us define our target phenotype and its tissure to start our GWAS
<div class="language-text highlight"><pre><span></span><code><span id="__span-0-1"><a id="__codelineno-0-1" name="__codelineno-0-1" href="#__codelineno-0-1"></a># Define the target phenotype and the affacted tissue
</span><span id="__span-0-2"><a id="__codelineno-0-2" name="__codelineno-0-2" href="#__codelineno-0-2"></a>target=&quot;atopy&quot;
</span><span id="__span-0-3"><a id="__codelineno-0-3" name="__codelineno-0-3" href="#__codelineno-0-3"></a>tissue=&quot;skin&quot;
</span><span id="__span-0-4"><a id="__codelineno-0-4" name="__codelineno-0-4" href="#__codelineno-0-4"></a>
</span><span id="__span-0-5"><a id="__codelineno-0-5" name="__codelineno-0-5" href="#__codelineno-0-5"></a># Let us explore how the data table looks like
</span><span id="__span-0-6"><a id="__codelineno-0-6" name="__codelineno-0-6" href="#__codelineno-0-6"></a>less phenotypes/conditions_${tissue}.csv
</span></code></pre></div></p>
<h2 id="512-identify-affected-cases-until-last-year">5.1.2. Identify affected cases until last year</h2>
<p>Most of the input files are designed to have 2 rows for each new year of study. The 1st row records the new phenotypes diagnosed only in that study year while the 2nd row records all phenotypes diagnosed from baseline through that study year. The column <code>to_date</code> is encoded 0 for the former and 1 for the latter. More can be read about the design of each table in the <a href="https://datacommons.morrisanimalfoundation.org/">Data Commons</a> of Morris Animal Foundation. For this tutorial, we will identify the cases as the dogs that showed the phenotypes at any year of the study.</p>
<div class="language-text highlight"><pre><span></span><code><span id="__span-1-1"><a id="__codelineno-1-1" name="__codelineno-1-1" href="#__codelineno-1-1"></a># Define and create the GWAS output folder
</span><span id="__span-1-2"><a id="__codelineno-1-2" name="__codelineno-1-2" href="#__codelineno-1-2"></a>gwas=&quot;gwas_${target}&quot; &amp;&amp; mkdir -p &quot;$gwas&quot;
</span><span id="__span-1-3"><a id="__codelineno-1-3" name="__codelineno-1-3" href="#__codelineno-1-3"></a>
</span><span id="__span-1-4"><a id="__codelineno-1-4" name="__codelineno-1-4" href="#__codelineno-1-4"></a># Identify the 2nd row of the last year of study for each dog
</span><span id="__span-1-5"><a id="__codelineno-1-5" name="__codelineno-1-5" href="#__codelineno-1-5"></a>cat phenotypes/conditions_${tissue}.csv | awk &#39;BEGIN{FS=&quot;,&quot;}\
</span><span id="__span-1-6"><a id="__codelineno-1-6" name="__codelineno-1-6" href="#__codelineno-1-6"></a>    NR==1{print $0;next}{if ($5==1) {\
</span><span id="__span-1-7"><a id="__codelineno-1-7" name="__codelineno-1-7" href="#__codelineno-1-7"></a>    if (!($1 in max_year) || $3 &gt; max_year[$1]) {
</span><span id="__span-1-8"><a id="__codelineno-1-8" name="__codelineno-1-8" href="#__codelineno-1-8"></a>        max_year[$1] = $3;
</span><span id="__span-1-9"><a id="__codelineno-1-9" name="__codelineno-1-9" href="#__codelineno-1-9"></a>        line[$1] = $0;
</span><span id="__span-1-10"><a id="__codelineno-1-10" name="__codelineno-1-10" href="#__codelineno-1-10"></a>    }}}END{for (id in line) {print line[id];}}&#39; | tr &#39;,&#39; &#39;\t&#39; &gt; &quot;$gwas&quot;/conditions_${tissue}_lastYear.tab
</span></code></pre></div>
<h2 id="513-count-the-of-cases-of-each-phenotype">5.1.3. Count the of cases of each phenotype</h2>
<div class="language-text highlight"><pre><span></span><code><span id="__span-2-1"><a id="__codelineno-2-1" name="__codelineno-2-1" href="#__codelineno-2-1"></a># Get the number of columns in the header.
</span><span id="__span-2-2"><a id="__codelineno-2-2" name="__codelineno-2-2" href="#__codelineno-2-2"></a>num_columns=$(head -n 1 &quot;$gwas/conditions_${tissue}_lastYear.tab&quot; | awk -F&quot;\t&quot; &#39;{print NF}&#39;)
</span><span id="__span-2-3"><a id="__codelineno-2-3" name="__codelineno-2-3" href="#__codelineno-2-3"></a>
</span><span id="__span-2-4"><a id="__codelineno-2-4" name="__codelineno-2-4" href="#__codelineno-2-4"></a># Loop over each column from the third till the last and print the sum.
</span><span id="__span-2-5"><a id="__codelineno-2-5" name="__codelineno-2-5" href="#__codelineno-2-5"></a>for ((i=7; i&lt;=$num_columns; i++)); do
</span><span id="__span-2-6"><a id="__codelineno-2-6" name="__codelineno-2-6" href="#__codelineno-2-6"></a>    awk -v col=&quot;$i&quot; &#39;BEGIN{FS=OFS=&quot;\t&quot;}NR==1{dis=$col;next}{sum+=$col} END {print dis,sum}&#39; &quot;$gwas&quot;/conditions_${tissue}_lastYear.tab
</span><span id="__span-2-7"><a id="__codelineno-2-7" name="__codelineno-2-7" href="#__codelineno-2-7"></a>done &gt; &quot;$gwas&quot;/conditions_${tissue}.no_of_cases
</span></code></pre></div>
<h2 id="52-sample-selection">5.2. Sample selection</h2>
<h2 id="521-select-cases-of-a-target-phenotype">5.2.1 Select "cases" of a target phenotype</h2>
<div class="language-text highlight"><pre><span></span><code><span id="__span-3-1"><a id="__codelineno-3-1" name="__codelineno-3-1" href="#__codelineno-3-1"></a>Tcol=$(head -n 1 &quot;$gwas/conditions_${tissue}_lastYear.tab&quot; | awk -F&quot;\t&quot; -v pheno=&quot;$target&quot; &#39;{for (i=7; i&lt;=NF; i++){if($i==pheno)print i}}&#39;)
</span><span id="__span-3-2"><a id="__codelineno-3-2" name="__codelineno-3-2" href="#__codelineno-3-2"></a>awk -v Tcol=&quot;$Tcol&quot; &#39;BEGIN{FS=OFS=&quot;\t&quot;}NR==1{print;next}{if($Tcol==&quot;1&quot;)print}&#39; &quot;$gwas&quot;/conditions_${tissue}_lastYear.tab &gt; &quot;$gwas&quot;/conditions_${target}_lastYear.tab
</span><span id="__span-3-3"><a id="__codelineno-3-3" name="__codelineno-3-3" href="#__codelineno-3-3"></a>
</span><span id="__span-3-4"><a id="__codelineno-3-4" name="__codelineno-3-4" href="#__codelineno-3-4"></a>tail -n+2 &quot;$gwas&quot;/conditions_${target}_lastYear.tab | cut -f1 &gt; &quot;$gwas&quot;/${target}_cases.ids
</span></code></pre></div>
<h2 id="522-check-for-co-existing-conditions">5.2.2 Check for co-existing conditions</h2>
<p><div class="language-text highlight"><pre><span></span><code><span id="__span-4-1"><a id="__codelineno-4-1" name="__codelineno-4-1" href="#__codelineno-4-1"></a>for ((i=7; i&lt;=$num_columns; i++)); do
</span><span id="__span-4-2"><a id="__codelineno-4-2" name="__codelineno-4-2" href="#__codelineno-4-2"></a>    awk -v col=&quot;$i&quot; &#39;BEGIN{FS=OFS=&quot;\t&quot;}NR==1{dis=$col;next}{sum+=$col} END {if(sum)print dis,sum}&#39; &quot;$gwas&quot;/conditions_${target}_lastYear.tab
</span><span id="__span-4-3"><a id="__codelineno-4-3" name="__codelineno-4-3" href="#__codelineno-4-3"></a>done &gt; &quot;$gwas&quot;/conditions_${target}_lastYear_coexist.tab
</span></code></pre></div>
For now, no sample will be excluded from cases.</p>
<h2 id="523-identify-individuals-to-be-excluded-from-controls">5.2.3. Identify individuals to be excluded from controls</h2>
<div class="language-text highlight"><pre><span></span><code><span id="__span-5-1"><a id="__codelineno-5-1" name="__codelineno-5-1" href="#__codelineno-5-1"></a>grep &#39;allergic_reaction\|seasonal_allergy\|angioedema\|facial_edema\|.*_dermatitis\|r_o_atopy\|vaccine_reaction&#39; &quot;$gwas&quot;/conditions_${tissue}.no_of_cases &gt; &quot;$gwas&quot;/${target}.to_be_excluded.lst
</span><span id="__span-5-2"><a id="__codelineno-5-2" name="__codelineno-5-2" href="#__codelineno-5-2"></a>while read offTarget;do
</span><span id="__span-5-3"><a id="__codelineno-5-3" name="__codelineno-5-3" href="#__codelineno-5-3"></a>  offTcol=$(head -n 1 &quot;$gwas&quot;/conditions_${tissue}_lastYear.tab | awk -F&quot;\t&quot; -v pheno=&quot;$offTarget&quot; &#39;{for (i=7; i&lt;=NF; i++){if($i==pheno)print i}}&#39;)
</span><span id="__span-5-4"><a id="__codelineno-5-4" name="__codelineno-5-4" href="#__codelineno-5-4"></a>  awk -F&quot;\t&quot; -v Tcol=&quot;$Tcol&quot; -v offTcol=&quot;$offTcol&quot; &#39;NR&gt;1{if($Tcol!=&quot;1&quot; &amp;&amp; $offTcol==&quot;1&quot;)print $1}&#39; &quot;$gwas&quot;/conditions_${tissue}_lastYear.tab
</span><span id="__span-5-5"><a id="__codelineno-5-5" name="__codelineno-5-5" href="#__codelineno-5-5"></a>done &lt; &lt;(cut -f1 &quot;$gwas&quot;/${target}.to_be_excluded.lst) | sort | uniq &gt; &quot;$gwas&quot;/${target}.to_be_excluded.ids
</span></code></pre></div>
<h2 id="524-remove-the-excluded-samples-from-the-genotyping-dataset-include-the-oldies-if-not-testing-a-phenotype-with-high-mortality-in-young-age">5.2.4. Remove the excluded samples from the genotyping dataset (include the oldies if not testing a phenotype with high mortality in young age)</h2>
<p><div class="language-text highlight"><pre><span></span><code><span id="__span-6-1"><a id="__codelineno-6-1" name="__codelineno-6-1" href="#__codelineno-6-1"></a>cat &quot;$gwas&quot;/${target}.to_be_excluded.ids | grep -Fwf - AxiomGT1v2.filtered.fam &gt; &quot;$gwas&quot;/${target}.to_be_excluded.samples
</span><span id="__span-6-2"><a id="__codelineno-6-2" name="__codelineno-6-2" href="#__codelineno-6-2"></a>grep ^Oldies AxiomGT1v2.filtered.fam &gt;&gt; &quot;$gwas&quot;/${target}.to_be_excluded.samples
</span><span id="__span-6-3"><a id="__codelineno-6-3" name="__codelineno-6-3" href="#__codelineno-6-3"></a>
</span><span id="__span-6-4"><a id="__codelineno-6-4" name="__codelineno-6-4" href="#__codelineno-6-4"></a>plink2 --bfile AxiomGT1v2.filtered --chr-set 38 no-xy --allow-extra-chr \
</span><span id="__span-6-5"><a id="__codelineno-6-5" name="__codelineno-6-5" href="#__codelineno-6-5"></a>       --remove &quot;$gwas&quot;/${target}.to_be_excluded.samples --maf 0.01 \
</span><span id="__span-6-6"><a id="__codelineno-6-6" name="__codelineno-6-6" href="#__codelineno-6-6"></a>       --make-bed --output-chr &#39;chrM&#39; --out &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}
</span></code></pre></div>
There are 578651 variants and 2487 samples that passed filters and QC.</p>
<h2 id="525-exclusion-of-1st-degree-relatives">5.2.5. Exclusion of 1st degree relatives</h2>
<p>Again, the genotyping data will be used to assess sample-distances and thus identify sample duplications. 0.177 (the geometric mean of 0.25 and 0.125) will be used as cutoff for the KING kinship coefficients to identify 1st degree relatives.</p>
<p><div class="language-text highlight"><pre><span></span><code><span id="__span-7-1"><a id="__codelineno-7-1" name="__codelineno-7-1" href="#__codelineno-7-1"></a># First, identify possible related dogs.
</span><span id="__span-7-2"><a id="__codelineno-7-2" name="__codelineno-7-2" href="#__codelineno-7-2"></a>plink2 --bfile &quot;$gwas&quot;/AxiomGT1v2.filtered.${target} --chr-set 38 no-xy --allow-extra-chr \
</span><span id="__span-7-3"><a id="__codelineno-7-3" name="__codelineno-7-3" href="#__codelineno-7-3"></a>       --king-cutoff 0.177 \
</span><span id="__span-7-4"><a id="__codelineno-7-4" name="__codelineno-7-4" href="#__codelineno-7-4"></a>       --out &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.1st_degree_relatives
</span><span id="__span-7-5"><a id="__codelineno-7-5" name="__codelineno-7-5" href="#__codelineno-7-5"></a>
</span><span id="__span-7-6"><a id="__codelineno-7-6" name="__codelineno-7-6" href="#__codelineno-7-6"></a># Now, those relatives can be removed to avoid inflation of false associations in the GWAS.
</span><span id="__span-7-7"><a id="__codelineno-7-7" name="__codelineno-7-7" href="#__codelineno-7-7"></a>plink2 --bfile &quot;$gwas&quot;/AxiomGT1v2.filtered.${target} --chr-set 38 no-xy --allow-extra-chr \
</span><span id="__span-7-8"><a id="__codelineno-7-8" name="__codelineno-7-8" href="#__codelineno-7-8"></a>       --remove &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.1st_degree_relatives.king.cutoff.out.id \
</span><span id="__span-7-9"><a id="__codelineno-7-9" name="__codelineno-7-9" href="#__codelineno-7-9"></a>       --make-bed --output-chr &#39;chrM&#39; --out &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives
</span></code></pre></div>
There are 578651 variants and 1989 samples (980 females, 1009 males) that passed filters and QC.</p>
<h2 id="53-generate-the-phenotype-file">5.3. Generate the phenotype file</h2>
<p><div class="language-text highlight"><pre><span></span><code><span id="__span-8-1"><a id="__codelineno-8-1" name="__codelineno-8-1" href="#__codelineno-8-1"></a>awk &#39;FNR==NR{a[$1]=1;next}{if(!a[$2])print $1,$2,&quot;1&quot;;else print $1,$2,&quot;2&quot;;}&#39; &quot;$gwas&quot;/${target}_cases.ids &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.fam &gt; &quot;$gwas&quot;/${target}.pheno
</span><span id="__span-8-2"><a id="__codelineno-8-2" name="__codelineno-8-2" href="#__codelineno-8-2"></a>cat &quot;$gwas&quot;/${target}.pheno | cut -d&quot; &quot; -f3 | sort | uniq -c
</span></code></pre></div>
There are 317 cases and 1672 controls.</p>
<h2 id="54-linkage-disequilibrium-ld-analysis">5.4. Linkage disequilibrium (LD) analysis</h2>
<h2 id="541-ld-estimation">5.4.1 LD estimation</h2>
<ul>
<li>--r2 reports squared inter-variant allele count correlations.</li>
<li>--ld-window-r2    variant pairs with r2 values &lt; 0.2 are filtered out by default. This option is used to adjust this threshold.</li>
</ul>
<div class="language-text highlight"><pre><span></span><code><span id="__span-9-1"><a id="__codelineno-9-1" name="__codelineno-9-1" href="#__codelineno-9-1"></a>plink --bfile &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives --chr-set 38 no-xy --allow-extra-chr \
</span><span id="__span-9-2"><a id="__codelineno-9-2" name="__codelineno-9-2" href="#__codelineno-9-2"></a>      --r2 --ld-window-r2 0 --ld-window-kb 10000 --ld-window 100 \
</span><span id="__span-9-3"><a id="__codelineno-9-3" name="__codelineno-9-3" href="#__codelineno-9-3"></a>      --output-chr &#39;chrM&#39; --out &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.stats
</span></code></pre></div>
<p>Generate a histogram of r2.
<div class="language-text highlight"><pre><span></span><code><span id="__span-10-1"><a id="__codelineno-10-1" name="__codelineno-10-1" href="#__codelineno-10-1"></a>awk -v size=0.01 &#39;BEGIN{OFS=&quot;\t&quot;;bmin=bmax=0}{ b=int($7/size); a[b]++; bmax=b&gt;bmax?b:bmax; bmin=b&lt;bmin?b:bmin } END { for(i=bmin;i&lt;=bmax;++i) print i*size,(i+1)*size,a[i]/1 }&#39;  &lt;(tail -n+2 &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.stats.ld) &gt; &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.stats.ld.r2_histo ## 155780 sequential markers have complete linkage
</span></code></pre></div></p>
<p>Create an LD Decay Plot.
<div class="language-text highlight"><pre><span></span><code><span id="__span-11-1"><a id="__codelineno-11-1" name="__codelineno-11-1" href="#__codelineno-11-1"></a>cat &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.stats.ld | awk &#39;/NR==1/{print;next}{x=$5-$2;if(x&lt;100000)print}&#39; &gt; &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.stats.ld2
</span><span id="__span-11-2"><a id="__codelineno-11-2" name="__codelineno-11-2" href="#__codelineno-11-2"></a>
</span><span id="__span-11-3"><a id="__codelineno-11-3" name="__codelineno-11-3" href="#__codelineno-11-3"></a>Rscript -e &#39;args=(commandArgs(TRUE));require(ggplot2);&#39;\
</span><span id="__span-11-4"><a id="__codelineno-11-4" name="__codelineno-11-4" href="#__codelineno-11-4"></a>&#39;ld_data &lt;- read.table(paste(args[1],&quot;/AxiomGT1v2.filtered.&quot;,args[2],&quot;.noRelatives.stats.ld2&quot;,sep=&quot;&quot;), header = TRUE);&#39;\
</span><span id="__span-11-5"><a id="__codelineno-11-5" name="__codelineno-11-5" href="#__codelineno-11-5"></a>&#39;ld_data$Distance &lt;- abs(ld_data$BP_B - ld_data$BP_A);&#39;\
</span><span id="__span-11-6"><a id="__codelineno-11-6" name="__codelineno-11-6" href="#__codelineno-11-6"></a>&#39;average_r2 &lt;- aggregate(R2 ~ Distance, data = ld_data, mean);&#39;\
</span><span id="__span-11-7"><a id="__codelineno-11-7" name="__codelineno-11-7" href="#__codelineno-11-7"></a>&#39;jpeg(file = paste(args[1],&quot;/LD_Decay.&quot;,args[2],&quot;.jpg&quot;,sep=&quot;&quot;));&#39;\
</span><span id="__span-11-8"><a id="__codelineno-11-8" name="__codelineno-11-8" href="#__codelineno-11-8"></a>&#39;ggplot(average_r2, aes(x = Distance, y = R2)) + geom_line() + theme_minimal() + labs(x = &quot;Distance (bp)&quot;, y = &quot;r2&quot;);&#39;\
</span><span id="__span-11-9"><a id="__codelineno-11-9" name="__codelineno-11-9" href="#__codelineno-11-9"></a>&#39;dev.off();&#39; &quot;$gwas&quot; &quot;${target}&quot;
</span></code></pre></div></p>
<h2 id="542-ld-pruning">5.4.2 LD pruning</h2>
<p>For this tutorial, we will prune pairwise SNPs with R2 &gt;â€‰ 0.8 (using windows of 100kb and step size of 1 SNP)</p>
<div class="language-text highlight"><pre><span></span><code><span id="__span-12-1"><a id="__codelineno-12-1" name="__codelineno-12-1" href="#__codelineno-12-1"></a>plink2 --bfile &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives --chr-set 38 no-xy --allow-extra-chr \
</span><span id="__span-12-2"><a id="__codelineno-12-2" name="__codelineno-12-2" href="#__codelineno-12-2"></a>       --indep-pairwise 100kb 0.8 \
</span><span id="__span-12-3"><a id="__codelineno-12-3" name="__codelineno-12-3" href="#__codelineno-12-3"></a>       --output-chr &#39;chrM&#39; --out &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_lst ## 397962 of 578651 variants removed
</span><span id="__span-12-4"><a id="__codelineno-12-4" name="__codelineno-12-4" href="#__codelineno-12-4"></a>
</span><span id="__span-12-5"><a id="__codelineno-12-5" name="__codelineno-12-5" href="#__codelineno-12-5"></a>plink2 --bfile &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives --chr-set 38 no-xy --allow-extra-chr \
</span><span id="__span-12-6"><a id="__codelineno-12-6" name="__codelineno-12-6" href="#__codelineno-12-6"></a>       --extract &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_lst.prune.in \
</span><span id="__span-12-7"><a id="__codelineno-12-7" name="__codelineno-12-7" href="#__codelineno-12-7"></a>       --make-bed --output-chr &#39;chrM&#39; --out &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_prune ## 180689 variants remaining
</span></code></pre></div>
<h2 id="543-ld-re-evaluation">5.4.3 LD re-evaluation</h2>
<div class="language-text highlight"><pre><span></span><code><span id="__span-13-1"><a id="__codelineno-13-1" name="__codelineno-13-1" href="#__codelineno-13-1"></a>plink --bfile &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_prune --chr-set 38 no-xy --allow-extra-chr \
</span><span id="__span-13-2"><a id="__codelineno-13-2" name="__codelineno-13-2" href="#__codelineno-13-2"></a>      --r2 --ld-window-r2 0 --ld-window-kb 10000 --ld-window 100 \
</span><span id="__span-13-3"><a id="__codelineno-13-3" name="__codelineno-13-3" href="#__codelineno-13-3"></a>      --output-chr &#39;chrM&#39; --out &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_prune.stats
</span><span id="__span-13-4"><a id="__codelineno-13-4" name="__codelineno-13-4" href="#__codelineno-13-4"></a>
</span><span id="__span-13-5"><a id="__codelineno-13-5" name="__codelineno-13-5" href="#__codelineno-13-5"></a>
</span><span id="__span-13-6"><a id="__codelineno-13-6" name="__codelineno-13-6" href="#__codelineno-13-6"></a># New histogram.
</span><span id="__span-13-7"><a id="__codelineno-13-7" name="__codelineno-13-7" href="#__codelineno-13-7"></a>awk -v size=0.01 &#39;BEGIN{OFS=&quot;\t&quot;;bmin=bmax=0}{ b=int($7/size); a[b]++; bmax=b&gt;bmax?b:bmax; bmin=b&lt;bmin?b:bmin } END { for(i=bmin;i&lt;=bmax;++i) print i*size,(i+1)*size,a[i]/1 }&#39;  &lt;(tail -n+2 &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_prune.stats.ld) &gt; &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_prune.stats.ld.r2_histo
</span><span id="__span-13-8"><a id="__codelineno-13-8" name="__codelineno-13-8" href="#__codelineno-13-8"></a>
</span><span id="__span-13-9"><a id="__codelineno-13-9" name="__codelineno-13-9" href="#__codelineno-13-9"></a># Compare the change.
</span><span id="__span-13-10"><a id="__codelineno-13-10" name="__codelineno-13-10" href="#__codelineno-13-10"></a>paste &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.stats.ld.r2_histo &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_prune.stats.ld.r2_histo | less
</span></code></pre></div>
<p>How does the LD Decay Plot look?
<div class="language-text highlight"><pre><span></span><code><span id="__span-14-1"><a id="__codelineno-14-1" name="__codelineno-14-1" href="#__codelineno-14-1"></a>cat &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_prune.stats.ld | awk &#39;/NR==1/{print;next}{x=$5-$2;if(x&lt;100000)print}&#39; &gt; &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_prune.stats.ld2
</span><span id="__span-14-2"><a id="__codelineno-14-2" name="__codelineno-14-2" href="#__codelineno-14-2"></a>
</span><span id="__span-14-3"><a id="__codelineno-14-3" name="__codelineno-14-3" href="#__codelineno-14-3"></a>Rscript -e &#39;args=(commandArgs(TRUE));require(ggplot2);&#39;\
</span><span id="__span-14-4"><a id="__codelineno-14-4" name="__codelineno-14-4" href="#__codelineno-14-4"></a>&#39;ld_data &lt;- read.table(paste(args[1],&quot;/AxiomGT1v2.filtered.&quot;,args[2],&quot;.noRelatives.LD_prune.stats.ld2&quot;,sep=&quot;&quot;), header = TRUE);&#39;\
</span><span id="__span-14-5"><a id="__codelineno-14-5" name="__codelineno-14-5" href="#__codelineno-14-5"></a>&#39;ld_data$Distance &lt;- abs(ld_data$BP_B - ld_data$BP_A);&#39;\
</span><span id="__span-14-6"><a id="__codelineno-14-6" name="__codelineno-14-6" href="#__codelineno-14-6"></a>&#39;average_r2 &lt;- aggregate(R2 ~ Distance, data = ld_data, mean);&#39;\
</span><span id="__span-14-7"><a id="__codelineno-14-7" name="__codelineno-14-7" href="#__codelineno-14-7"></a>&#39;jpeg(file = paste(args[1],&quot;/LD_Decay_after_prune.&quot;,args[2],&quot;.jpg&quot;,sep=&quot;&quot;));&#39;\
</span><span id="__span-14-8"><a id="__codelineno-14-8" name="__codelineno-14-8" href="#__codelineno-14-8"></a>&#39;ggplot(average_r2, aes(x = Distance, y = R2)) + geom_line() + theme_minimal() + labs(x = &quot;Distance (bp)&quot;, y = &quot;r2&quot;);&#39;\
</span><span id="__span-14-9"><a id="__codelineno-14-9" name="__codelineno-14-9" href="#__codelineno-14-9"></a>&#39;dev.off();&#39; &quot;$gwas&quot; &quot;${target}&quot;
</span></code></pre></div></p>
<h2 id="55-pca-to-identify-population-admixture-and-to-assess-distribution-of-cases">5.5. PCA to identify population admixture and to assess distribution of cases</h2>
<h2 id="551-calculation-of-pca">5.5.1. Calculation of PCA</h2>
<p>The <code>--pca</code> function in Plink2 extracts top principal components from the variance-standardized relationship matrix (i.e. genomic relationship matrix). LD pruning is adviced before PCA to reduce the risk of getting PCs based on just a few genomic regions. This behaviour tends to prevent deflation of statistical testing results later.</p>
<div class="language-text highlight"><pre><span></span><code><span id="__span-15-1"><a id="__codelineno-15-1" name="__codelineno-15-1" href="#__codelineno-15-1"></a>plink2 --bfile &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_prune --chr-set 38 no-xy --allow-extra-chr \
</span><span id="__span-15-2"><a id="__codelineno-15-2" name="__codelineno-15-2" href="#__codelineno-15-2"></a>       --autosome --pca \
</span><span id="__span-15-3"><a id="__codelineno-15-3" name="__codelineno-15-3" href="#__codelineno-15-3"></a>       --output-chr &#39;chrM&#39; --out &quot;$gwas&quot;/AxiomGT1v2.filtered.${target}.noRelatives.LD_prune.pca
</span></code></pre></div>
<p>The variance can be plotted to identify the principal components that explain most of the variance.
<div class="language-text highlight"><pre><span></span><code><span id="__span-16-1"><a id="__codelineno-16-1" name="__codelineno-16-1" href="#__codelineno-16-1"></a>Rscript -e &#39;args=(commandArgs(TRUE));&#39;\
</span><span id="__span-16-2"><a id="__codelineno-16-2" name="__codelineno-16-2" href="#__codelineno-16-2"></a>&#39;val &lt;- read.table(paste(args[1],&quot;/AxiomGT1v2.filtered.&quot;,args[2],&quot;.noRelatives.LD_prune.pca.eigenval&quot;,sep=&quot;&quot;));&#39;\
</span><span id="__span-16-3"><a id="__codelineno-16-3" name="__codelineno-16-3" href="#__codelineno-16-3"></a>&#39;val$varPerc &lt;- val$V1/sum(val$V1);&#39;\
</span><span id="__span-16-4"><a id="__codelineno-16-4" name="__codelineno-16-4" href="#__codelineno-16-4"></a>&#39;jpeg(file = paste(args[1],&quot;/Var_PCs.jpg&quot;,sep=&quot;&quot;));&#39;\
</span><span id="__span-16-5"><a id="__codelineno-16-5" name="__codelineno-16-5" href="#__codelineno-16-5"></a>&#39;plot( x = seq(1:length(val$varPerc)), y = val$varPerc, type = &quot;o&quot;,xlab = &quot;principal Component&quot;, ylab = &quot;Variance explained in %&quot;);&#39;\
</span><span id="__span-16-6"><a id="__codelineno-16-6" name="__codelineno-16-6" href="#__codelineno-16-6"></a>&#39;dev.off();&#39; &quot;$gwas&quot; &quot;${target}&quot;
</span></code></pre></div></p>
<p><a class="glightbox" href="../images/Var_PCs.jpg" data-type="image" data-width="auto" data-height="auto" data-desc-position="bottom"><img alt="" src="../images/Var_PCs.jpg" /></a><!-- --></p>
<h2 id="552-assess-possible-batch-effect">5.5.2. Assess possible batch effect</h2>
<p>Plot the main principal components and color by the study batches:
<div class="language-text highlight"><pre><span></span><code><span id="__span-17-1"><a id="__codelineno-17-1" name="__codelineno-17-1" href="#__codelineno-17-1"></a>Rscript -e &#39;args=(commandArgs(TRUE));require(ggplot2);require(gridExtra);&#39;\
</span><span id="__span-17-2"><a id="__codelineno-17-2" name="__codelineno-17-2" href="#__codelineno-17-2"></a>&#39;eigenvec &lt;- read.table(paste(args[1],&quot;/AxiomGT1v2.filtered.&quot;,args[2],&quot;.noRelatives.LD_prune.pca.eigenvec&quot;,sep=&quot;&quot;), header = TRUE, comment.char=&quot;&quot;);&#39;\
</span><span id="__span-17-3"><a id="__codelineno-17-3" name="__codelineno-17-3" href="#__codelineno-17-3"></a>&#39;eigenvec$X.FID &lt;-  as.factor(eigenvec$X.FID);&#39;\
</span><span id="__span-17-4"><a id="__codelineno-17-4" name="__codelineno-17-4" href="#__codelineno-17-4"></a>&#39;plot1 &lt;- ggplot(eigenvec, aes(x = PC1, y = PC2, col = X.FID)) + geom_point() + labs(title = &quot;PCA Plot&quot;, x = &quot;PC1&quot;, y = &quot;PC2&quot;);&#39;\
</span><span id="__span-17-5"><a id="__codelineno-17-5" name="__codelineno-17-5" href="#__codelineno-17-5"></a>&#39;plot2 &lt;- ggplot(eigenvec, aes(x = PC3, y = PC4, col = X.FID)) + geom_point() + labs(title = &quot;PCA Plot&quot;, x = &quot;PC3&quot;, y = &quot;PC4&quot;);&#39;\
</span><span id="__span-17-6"><a id="__codelineno-17-6" name="__codelineno-17-6" href="#__codelineno-17-6"></a>&#39;plot3 &lt;- ggplot(eigenvec, aes(x = PC1, y = PC3, col = X.FID)) + geom_point() + labs(title = &quot;PCA Plot&quot;, x = &quot;PC1&quot;, y = &quot;PC3&quot;);&#39;\
</span><span id="__span-17-7"><a id="__codelineno-17-7" name="__codelineno-17-7" href="#__codelineno-17-7"></a>&#39;plot4 &lt;- ggplot(eigenvec, aes(x = PC2, y = PC3, col = X.FID)) + geom_point() + labs(title = &quot;PCA Plot&quot;, x = &quot;PC2&quot;, y = &quot;PC3&quot;);&#39;\
</span><span id="__span-17-8"><a id="__codelineno-17-8" name="__codelineno-17-8" href="#__codelineno-17-8"></a>&#39;combined_plot &lt;- grid.arrange(plot1, plot2, plot3, plot4, nrow = 2);&#39;\
</span><span id="__span-17-9"><a id="__codelineno-17-9" name="__codelineno-17-9" href="#__codelineno-17-9"></a>&#39;ggsave(paste(args[1],&quot;/pca_plot.png&quot;,sep=&quot;&quot;), combined_plot, width = 12, height = 8, dpi = 400);&#39; &quot;$gwas&quot; &quot;${target}&quot;
</span></code></pre></div></p>
<p><a class="glightbox" href="../images/pca_plot.png" data-type="image" data-width="auto" data-height="auto" data-desc-position="bottom"><img alt="" src="../images/pca_plot.png" /></a><!-- --></p>
<p>An additional tutorial of possible admixture analysis is <a href="https://morrisanimalfoundation.github.io/grGWAS/ext_docs/admixture_analysis/">here</a>.</p>
<h2 id="553-assess-distribution-of-cases-in-pca">5.5.3 Assess distribution of cases in PCA</h2>
<div class="language-text highlight"><pre><span></span><code><span id="__span-18-1"><a id="__codelineno-18-1" name="__codelineno-18-1" href="#__codelineno-18-1"></a>Rscript -e &#39;args=(commandArgs(TRUE));require(ggplot2);require(gridExtra);&#39;\
</span><span id="__span-18-2"><a id="__codelineno-18-2" name="__codelineno-18-2" href="#__codelineno-18-2"></a>&#39;eigenvec &lt;- read.table(paste(args[1],&quot;/AxiomGT1v2.filtered.&quot;,args[2],&quot;.noRelatives.LD_prune.pca.eigenvec&quot;,sep=&quot;&quot;), header = TRUE, comment.char=&quot;&quot;);&#39;\
</span><span id="__span-18-3"><a id="__codelineno-18-3" name="__codelineno-18-3" href="#__codelineno-18-3"></a>&#39;pheno &lt;- read.table(paste(args[1],&quot;/&quot;,args[2],&quot;.pheno&quot;,sep=&quot;&quot;), header = FALSE, comment.char=&quot;&quot;);&#39;\
</span><span id="__span-18-4"><a id="__codelineno-18-4" name="__codelineno-18-4" href="#__codelineno-18-4"></a>&#39;names(pheno) &lt;- c(&quot;FID&quot;, &quot;IID&quot;, &quot;pheno&quot;);&#39;\
</span><span id="__span-18-5"><a id="__codelineno-18-5" name="__codelineno-18-5" href="#__codelineno-18-5"></a>&#39;pheno$pheno[pheno$pheno == &quot;1&quot;] &lt;- &quot;Unaffected&quot;;pheno$pheno[pheno$pheno == &quot;2&quot;] &lt;- &quot;Affected&quot;;&#39;\
</span><span id="__span-18-6"><a id="__codelineno-18-6" name="__codelineno-18-6" href="#__codelineno-18-6"></a>&#39;ph_eigenvec &lt;-  merge(pheno, eigenvec, by = &quot;IID&quot;);&#39;\
</span><span id="__span-18-7"><a id="__codelineno-18-7" name="__codelineno-18-7" href="#__codelineno-18-7"></a>&#39;ph_eigenvec$pheno &lt;-  as.factor(ph_eigenvec$pheno);&#39;\
</span><span id="__span-18-8"><a id="__codelineno-18-8" name="__codelineno-18-8" href="#__codelineno-18-8"></a>&#39;plot1 &lt;- ggplot(ph_eigenvec, aes(x = PC1, y = PC2, col = pheno)) + geom_point() + labs(title = &quot;PCA Plot&quot;, x = &quot;PC1&quot;, y = &quot;PC2&quot;);&#39;\
</span><span id="__span-18-9"><a id="__codelineno-18-9" name="__codelineno-18-9" href="#__codelineno-18-9"></a>&#39;plot2 &lt;- ggplot(ph_eigenvec, aes(x = PC3, y = PC4, col = pheno)) + geom_point() + labs(title = &quot;PCA Plot&quot;, x = &quot;PC3&quot;, y = &quot;PC4&quot;);&#39;\
</span><span id="__span-18-10"><a id="__codelineno-18-10" name="__codelineno-18-10" href="#__codelineno-18-10"></a>&#39;plot3 &lt;- ggplot(ph_eigenvec, aes(x = PC1, y = PC3, col = pheno)) + geom_point() + labs(title = &quot;PCA Plot&quot;, x = &quot;PC1&quot;, y = &quot;PC3&quot;);&#39;\
</span><span id="__span-18-11"><a id="__codelineno-18-11" name="__codelineno-18-11" href="#__codelineno-18-11"></a>&#39;plot4 &lt;- ggplot(ph_eigenvec, aes(x = PC2, y = PC3, col = pheno)) + geom_point() + labs(title = &quot;PCA Plot&quot;, x = &quot;PC2&quot;, y = &quot;PC3&quot;);&#39;\
</span><span id="__span-18-12"><a id="__codelineno-18-12" name="__codelineno-18-12" href="#__codelineno-18-12"></a>&#39;combined_plot &lt;- grid.arrange(plot1, plot2, plot3, plot4, nrow = 2);&#39;\
</span><span id="__span-18-13"><a id="__codelineno-18-13" name="__codelineno-18-13" href="#__codelineno-18-13"></a>&#39;ggsave(paste(args[1],&quot;/pca_plot_&quot;,args[2],&quot;.png&quot;,sep=&quot;&quot;), combined_plot, width = 12, height = 8, dpi = 400);&#39; &quot;$gwas&quot; &quot;${target}&quot;
</span></code></pre></div>
<p><a class="glightbox" href="../images/pca_plot_atopy.jpg" data-type="image" data-width="auto" data-height="auto" data-desc-position="bottom"><img alt="" src="../images/pca_plot_atopy.jpg" /></a><!-- --></p>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
      
      <nav class="md-footer__inner md-grid" aria-label="Footer" >
        
          
          <a href="../4.qc_filter/" class="md-footer__link md-footer__link--prev" aria-label="Previous: 4. QC &amp; filtration">
            <div class="md-footer__button md-icon">
              
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
            </div>
            <div class="md-footer__title">
              <span class="md-footer__direction">
                Previous
              </span>
              <div class="md-ellipsis">
                4. QC & filtration
              </div>
            </div>
          </a>
        
        
          
          <a href="../6.heritability/" class="md-footer__link md-footer__link--next" aria-label="Next: 6. Heritability">
            <div class="md-footer__title">
              <span class="md-footer__direction">
                Next
              </span>
              <div class="md-ellipsis">
                6. Heritability
              </div>
            </div>
            <div class="md-footer__button md-icon">
              
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>
            </div>
          </a>
        
      </nav>
    
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["content.code.copy", "content.code.annotate", "content.action.view", "content.tooltips", "navigation.footer", "navigation.indexes", "navigation.sections", "navigation.tabs", "search.highlight", "content.tabs.link"], "search": "../assets/javascripts/workers/search.c011b7c0.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.7389ff0e.min.js"></script>
      
    
  <script>document$.subscribe(() => {const lightbox = GLightbox({"touchNavigation": true, "loop": false, "zoomable": true, "draggable": true, "openEffect": "zoom", "closeEffect": "zoom", "slideEffect": "slide"});})</script></body>
</html>